x-pack/legacy/plugins/actions/server/builtin_action_types/slack.test.ts

```diff
@@ -18,47 +17,19 @@ const services: Services = {
   savedObjectsClient: savedObjectsClientMock.create(),
 };
 
-let actionTypeRegistry: ActionTypeRegistry;
 let actionType: ActionType;
 
-async function mockSlackExecutor(options: ActionTypeExecutorOptions): Promise<any> {
-  const { params } = options;
-  const { message } = params;
-  if (message == null) throw new Error('message property required in parameter');
-
-  const failureMatch = message.match(/^failure: (.*)$/);
-  if (failureMatch != null) {
-    const failMessage = failureMatch[1];
-    throw new Error(`slack mockExecutor failure: ${failMessage}`);
-  }
-
-  return {
-    text: `slack mockExecutor success: ${message}`,
-  };
-}
-
 beforeAll(() => {
-  actionTypeRegistry = new ActionTypeRegistry({
-    taskManager: taskManagerMock.create(),
-    taskRunnerFactory: new TaskRunnerFactory(new ActionExecutor()),
-  });
-  actionTypeRegistry.register(getActionType({ executor: mockSlackExecutor }));
-  actionType = actionTypeRegistry.get(ACTION_TYPE_ID);
-
-  test('ensure action type is valid', () => {
-    expect(actionType).toBeTruthy();
+  actionType = getActionType({
+    async executor(options: ActionTypeExecutorOptions): Promise<any> {},
+    configurationUtilities: configUtilsMock,
   });
 });
 
-describe('action is registered', () => {
-  test('gets registered with builtin actions', () => {
```

Comment:
This is essentially an integration test and is covered by the function test suite